packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -19,7 +20,7 @@ async function getValidOriginsListFromCDN(): Promise<string[]> {
     return validOriginsCache;
   }
   if (!inServerSideRenderingEnvironment()) {
-    return fetch(validOriginsCdnEndpoint)
+    return fetch(validOriginsCdnEndpoint, { signal: AbortSignal.timeout(ORIGIN_LIST_TIMEOUT) })
```

Comment:
Can we add logging, before `fetch` is called, to indicate that we're about to make the CDN call?